docs/framework/development-tools/package-generator/javascript-package.md

```diff
@@ -124,7 +130,22 @@ Examples:
 npm run stylelint
 ```
 
-### `dll:build`
+### `build:dist`
+
+Creates the `NPM` and browser builds which can be loaded into an editor using TODO: LINK/NEW GUIDE??.
```

Comment:
```suggestion
Creates the npm and browser builds which can be loaded into an editor using TODO: LINK/NEW GUIDE??.
```

1. npm is lowercase.
2. I think npm should be in code quotes (`` `npm` ``) only if we are talking about npm command like `npm run xxx`